.github/workflows/binaries-and-deb-release.yml

```
@@ -0,0 +1,522 @@
+name: Build, attest, and sign standalone binaries plus Debian packages
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: "Release version tag (e.g. v1.2.3)"
+        required: true
+        type: string
+      git-sha:
+        description: "Source commit SHA"
+        required: true
+        type: string
+      oci-artifact-name:
+        description: "Name of the exported binaries artifact to consume"
+        required: true
+        type: string
+      platforms:
+        description: "Comma- or newline-separated list of linux/ARCH platforms to package"
+        required: false
+        default: linux/amd64
+        type: string
+      smoke-distro:
+        description: "Debian release used for binary/package smoke tests"
+        required: false
+        default: bullseye
+        type: string
+      apt-distributions:
+        description: "Comma- or newline-separated Debian distributions to mirror/publish to apt.z.cash"
+        required: false
+        default: bullseye,bookworm,trixie
```

Ditto (and here the default doesn't match `release.yml`, it includes `trixie` which is listed there as a TODO implying it doesn't work yet):
```suggestion
        required: true
```